test/distributed/fsdp/test_fsdp_sharded_grad_scaler.py

```diff
@@ -196,6 +204,153 @@ def test_fsdp_ddp_parity_with_grad_scaler(
                 sharded_grad_scaler_kwargs=sharded_grad_scaler_kwargs,
             )
 
+    def _build_model_and_optim(
+        self,
+        build_extra_ddp: bool = False,
```

Comment:
nit: If you always pass in the args into `_build_model_and_optim()` (which is the case this time), then we may not want to use default args since they could be error-prone.

Also, we do not need to include the `build_extra_ddp` complexity here if we are always passing `build_extra_ddp=True`. If we find that we need it in the future, then we can add it then.